internal/appconfig/setters.go

```diff
@@ -261,13 +261,13 @@ func (c *Config) SetStatics(statics []Static) {
 	}
 }
 
-func (c *Config) SetVolumes(volumes []Volume) {
+func (c *Config) SetVolumes(volumes []Mount) {
 	c.RawDefinition["mounts"] = volumes
 	// FIXME: "mounts" section is confusing, it is plural but only allows one mount
 	if len(volumes) > 0 {
```

Comment:
Should this just set `c.Mounts` to `volumes` directly instead of pulling the first one?

Reply:
good call! 34480705